### ISSUE TITLE
Improve accessibility of FreeCEN Newsletters — PDFs

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeCENMigration#1861]
Now that the Newsletters are being published for all users to read, we need to do as much as we can to make them accessible to all users, including screen reader users.

Unless a PDF is made to be accessible it won't be.

We are limited by the software that is available to us. Chris Mahoney cannot get LibreOffice (first choice) to work on his computer, so we are doing our best with OpenOffice (second choice). Initial testing is being done with VoiceOver in Preview on a Mac — Preview beats Safari and Acrobat Reader. 

Chris is using a checker to look for typos and similar errors.

We are using a scoring scheme to roughly assess progress as we learn to make the best accessible PDFs in OpenOffice.

We could apply to have a [Grackle Docs](https://pdfa.org/co-ceo/) assessment in due course, maybe once all OpenOffice changes are in place.

**Comment by AlOneill** [comment]
- guide written for OpenOffice
- score improved from May to June, frm 29% to 52%

**Comment by DeniseColbert** [comment]
Part of BAU so closing

**Comment by AlOneill** [comment]
@DeniseColbert 
I'm not sure why this has been closed – I do not know what BAU stands for. 

There are further improvements to be made, but I need to discuss these with Chris first and get his agreement: some changes rely on his editing of the new pieces table; other changes risk alienating contributors. To give you an idea of what still needs to be tackled:
- keyboard-only access is rubbish – links are not reachable or operable
- table accessibility could be improved with a caption and the removal of the empty row
- image accessibility needs improvement
- we need a strategy to encourage writing styles that are more accessible

**Comment by AlOneill** [comment]
@DeniseColbert 
Does BAU stand for beyond and above? If so, I don't think we have reached that point quite yet, unless you know something that I don't.

We probably won't be able to achieve link accessibility for keyboard-only users, but the other items are probably within reach.

**Comment by AlOneill** [comment]
@DeniseColbert 
Of course, BAU doesn't stand for beyond and above. What was I thinking? Anyway, my comment still stands.

**Comment by AlOneill** [comment]
Text tweaked on Newsletters page to reflect:
1. the difficulty that Windows users are experiencing with operating the external links in the Newsletters in a PDF viewer (details limited)
2. the updated options on the Contact form which now include "Newsletter"

**Comment by DeniseColbert** [comment]
Seems to be BAU now, closing. Can reopen (or make new story) if needed.

**Comment by AlOneill** [comment]
I still do not know what BAU stands for. Not knowing, I cannot agree that we have done all that is possible using OpenOffice, so I will re-open.

To be done:
- complete work on making table accessible
- complete work on making images accessible
- improve accessibility of text, especially recurring items

**Comment by DeniseColbert** [comment]
@AlOneill BAU stands for business as usual, apologies for using the abbreviation. 

If there's more to do, as you say, let's leave it open so we can track.

**Comment by Vino-S** [comment]
@AlOneill Should we close this issue?

**Comment by AlOneill** [comment]
@Vino-S 

No, do not close: we have a way to go yet. Problems keep arising.

**Comment by stephencoward-source** [comment]
Currently being worked on. 
Chase Chris via email for update. 15/06
